packages/x-adapter-platform/src/types/responses/models/result.model.ts

```diff
@@ -0,0 +1,18 @@
+import { PlatformTagging } from './tagging.model';
+
+/**
+ * Result model for the `platform` API.
+ *
+ * @public
+ */
+export interface PlatformResult {
```

Comment:
Where did this interface came from? It is different from the platform demo environment.

Reply:
We just moved it to this file but didn't touch the model. I think it is based on the platform demo environment, but I have to agree that the model differs from it.